pkg/resources/account_parameter_acceptance_test.go

```diff
@@ -48,19 +48,3 @@ func TestAcc_AccountParameter_PREVENT_LOAD_FROM_INLINE_URL(t *testing.T) {
 		},
 	})
 }
-
-func TestAcc_AccountParameter_CLIENT_METADATA_USE_SESSION_DATABASE(t *testing.T) {
-	resource.ParallelTest(t, resource.TestCase{
-		Providers:    providers(),
-		CheckDestroy: nil,
-		Steps: []resource.TestStep{
-			{
-				Config: accountParameterBasic("CLIENT_METADATA_USE_SESSION_DATABASE", "true"),
-				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttr("snowflake_account_parameter.p", "key", "CLIENT_METADATA_USE_SESSION_DATABASE"),
-					resource.TestCheckResourceAttr("snowflake_account_parameter.p", "value", "true"),
-				),
-			},
-		},
-	})
-}
```

Comment:
Why are we removing this test? It should still pass, right?

Reply:
This test tries to set CLIENT_METADATA_USE_SESSION_DATABASE for account.
CLIENT_METADATA_USE_SESSION_DATABASE is a session parameter and I think we should only allow to set it using a session_parameter. For now I modified it to use an actual account parameter, but if you think we should allow to set session parameters using account_parameter resource then we can discuss it